### DISCUSSION TITLE
Specify Windows certificate store

### DISCUSSION THREAD

**Discussion by Dennis276** [microsoft/msquic#1442] [Q&A] [answered]
Is there a way to specify which Windows certificate store msquic searches for the certificate? It seems like it only searches the user certificate store. I am currently using the msquic library in a Windows service running under the SYSTEM account, and would therefore like to be able to use the Local Computer certificate store.
Or am I missing something obvious?

**Comment by nibanks** [comment, marked as answer]
You're going to want to take a look at `QUIC_CERTIFICATE_HASH_STORE` which allows you to specify the store name and some `QUIC_CERTIFICATE_HASH_STORE_FLAGS ` flags, such as `QUIC_CERTIFICATE_HASH_STORE_FLAG_MACHINE_STORE` since you're dealing with machine, not user certificates.
